deno/README.md

```diff
@@ -0,0 +1,72 @@
+# Flatpak Deno Generator
+
+Run from jsr
+
+```
+deno -RN -W=. jsr:@flatpak-contrib/flatpak-deno-generator deno.lock
+```
+
+or locally from this repo
+
+```
+deno -RN -W=. main.ts deno.lock --output sources.json
```

Comment:
this only works from inside the src folder, can you add cd deno/src?

Reply:
I changed it to src/main.ts